src/main/java/uk/gov/hmcts/reform/professionalapi/repository/ProfessionalUserRepository.java

```diff
@@ -32,6 +32,16 @@ public interface ProfessionalUserRepository extends JpaRepository<ProfessionalUs
             nativeQuery = true)
     int findByUserCountByOrganisationId(@Param("organisationId") UUID organisationId);
 
+    @Query("SELECT pu FROM professional_user pu WHERE "
+        + "(COALESCE(:organisationIdentifiers) IS NULL OR pu.organisation.organisationIdentifier "
+        + "IN :organisationIdentifiers) and "
+        + "((:organisationId IS NULL OR CAST(pu.organisation.id AS string) > :organisationId) OR "
+        + "(:userId IS NULL OR CAST(pu.id AS string) > :userId)) ORDER BY "
+        + "CAST(pu.organisation.id AS string), CAST(pu.id AS string)")
+    Page<ProfessionalUser> findUsersInOrganisations(
+        @Param("organisationIdentifiers") List<String> organisationIdentifiers,
```

Comment:
I remembered why I have the IDs a String type instead of a UUID. When I revert back to UUID JPL throws an exception because it can't handle the UUID when running the query
`did not match expected type [java.lang.String (n/a)]`